<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2dVerificationWindowSettingWidget</class>
 <widget class="QWidget" name="Graph2dVerificationWindowSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelLabel">
           <property name="text">
            <string>Label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="labelLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="yAxisGroupBox">
         <property name="title">
          <string>Y Axis</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="yAxisLeftRadioButton">
            <property name="text">
             <string>Left</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="yAxisRightRadioButton">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="displayStyleGroupBox">
       <property name="title">
        <string>Display Style</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="displayStyleLineRadioButton">
          <property name="text">
           <string>Line</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="displayStyleSymbolRadioButton">
          <property name="text">
           <string>Symbol</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lineStyleLabel">
       <property name="text">
        <string>Line Style:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="lineStyleComboBox">
       <item>
        <property name="text">
         <string>Solid Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dashed Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dotted Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dots and Dashes</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lineWidthLabel">
       <property name="text">
        <string>Line Width:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="lineWidthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="barChartLabel">
       <property name="text">
        <string>Bar Chart:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QCheckBox" name="barChartCheckBox">
       <property name="text">
        <string>Show as Bar Chart</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="symbolStyleLabel">
       <property name="text">
        <string>Symbol Style:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QComboBox" name="symbolStyleComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Circle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rectangle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Diamond</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cross</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>XCross</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="ColorEditWidget" name="colorWidget">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="idWidget">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>displayStyleLineRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineStyleComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayStyleLineRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineWidthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayStyleSymbolRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>symbolStyleComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayStyleLineRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>barChartCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
